### ISSUE TITLE
Ship capturing 

### ISSUE THREAD

**Issue by KyroFirehammer** [endless-sky/endless-sky#2947]
So I'm trying to cap a world ship as I type this [it's the quest one ] bit why do they somehow if you leave the capture dialogue, land, restock and suddenly their right back up to min crew. This makes no sense at all and IMO should he fixed because I've been sitting here for 3 hours praying to RNGesus and even with the bet obtainable outfits for the job still am short crew for it.

**Comment by gunqqer** [comment]
It's for balance.

Because it would be too easy to cap everything. You could do it from start if you want to spam it.

**Comment by KyroFirehammer** [comment]
At least add a way to get crew from allied ships then restock. I've gotten this stupid AS down to 16 crew and lost I am getting soooooo close to editing my files

**Comment by gunqqer** [comment]
World ships arn't (as of 0.9.8) suposed to be able to captured. Just go find the plugin that gives OP skylance rifles. Until Quarg and Korath get reworked this won't ever, ever happen

**Comment by ghoulavenger** [comment]
Wasn't MZ considering allowing people to capture worldships in the future because of the removal of mass expansions in favor of outfits expansions? So I wouldn't say that capturing everything is necessarily the problem, although having a world ship would mean that you'd have to probably change the Quarg, since I haven't heard of any plans to allow people to capture Quarg ships as of yet (and worldships have like 3x the max crew of a bactrian outfit to full bunks by default). Sadly as of 0.9.8 world ships cannot be captured legitimately, but if you want to persist, you could lower the minimum crew of the worldship, raise the effectiveness of nerve gas (probably 2x is enough), or get that plugin gunqqer was referencing.

**Comment by KyroFirehammer** [comment]
I DID IT!!!!!!!!!! WOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO HYPE HYPE HYPE HAHA YEA BOI GET REKT "U CANT CAP A WS MORE LIKE U GUYS GOT BS!!!!!" that is not aimed at u at all btw I was talking bout my carrier idea to some dudes and they said I couldn't even cap a WS

**Comment by Hadron1776** [comment]
> why do they somehow if you leave the capture dialogue, land, restock and suddenly their right back up to min crew. 

This was done intentionally, because it was too easy to wear down the crews of mission ships over many days.

**Comment by gunqqer** [comment]
Take. This. To. Somewhere. Else.

This is *not* the place for that kind of thing, Kyro. There are 98 people who get an email message every time someone does something on this github page. I am one of them. Please close this issue as this will not be changed for sometime. 0.9.9 won't containt this change unless capping gets a compleat overhaul in it.

**Comment by KyroFirehammer** [comment]
Sorry.... The capping system needs an update though.

**Comment by Bladewood** [comment]
This was an intentional change, because it made capturing Quarg ships and Arfectas trivial.

**Comment by Startome** [comment]
^

**Comment by MessyMix** [comment]
>The capping system needs an update though.

@KyroFirehammer your attitude here needs an update though.

I don't know how many times you've been told to keep it more PROPER, post less SPAM and stop acting like a 10 year old. Which might be the case, so nothing against you, but if you want to roll with the big boys you gotta act like one. Please. Proper english. No CAPS LOCK YEA BOI GET REKT HYPE SHIT, okay? Just because you sat around for 3 hours trying to cap a worldship (savescumming so hard you might as well have cheated yourself one), doesn't mean you should be POSTING IT ON GITHUB LIKE THIS HYPE HYPE YEAH REKT BOI.

You can be more mature, I know it. Let's try.

**Comment by KiLEdEnNis** [comment]
this does not mean we hate you or hold a grudge or anything; if you just change your attitude to be more professional like MessyMix said we'll be happy to have you with us on GitHub (I started immature to a lesser extend as well, takes time to get used to GitHub)

**Comment by KyroFirehammer** [comment]
Fair enough, but I've dicided that I'm also going to try to see if I can't rebalance the WS (lower crew, and up the engine and weapon space a bit ) because it does make the game way easier once you have one as it currently stands (and to hard to get but it should still be difficult).

**Comment by KiLEdEnNis** [comment]
personally I'd not touch the World Ship and let MZ do it himself, since he was planning on making it capturable himself
but you can try

**Comment by KyroFirehammer** [comment]
I mean if that's what he's working on i don't want to mess up what he is doing. On a related note I made a deal discussion about an idea for another story arc I have omly posted part 1 out of 4 though. The will be new ships but this is intended to be super t3 almost I want to say t4 but I'm not even sure that's a thing. I'm going to wait on making it in code because it kinda puts the quarg in their place and makes the drak look less of a all powerful species. I'm planning on a whole set of t3 ships made by lower end species (sorta playing off the idea of the arfecta and PDT(point defence turret) where a normally lower end species has a really good ship. [ like the human governments each get a t3 ship, and the sundcate get some nice t1.1 ships (their version of a dreadnought or cruiser)

**Comment by KiLEdEnNis** [comment]
1. the Pug aren't a lower end species w/ a really good ship; they're apparently around Drak power, and they happen to have weak ships intentionally
2. if ya work on your content do it on the google groups or Discord

**Comment by Lineth** [comment]
The Pug are at _least_ equal to the Drak. They are implied to have fought against Archons and won, and certainly defeated the Quarg thoroughly enough for the Quarg to be scared of them. The only reason humans won the war against them was the Pug let them win by limiting themselves to T1 ships. It's a fairly major plot point about them. Even the Arfecta is likely the Pug showing restraint.

**Comment by MessyMix** [comment]
It is implied that Drak ≠ Archon, so just because they fought Archon and won, doesn't mean they are "at least equal to the Drak", although I have no doubt they are.

But, where is it implied that they fought and defeated Archon? I wouldn't be surprised if they could, but is this conveyed in the game?

**Comment by Lineth** [comment]
One of the parts hwere you conferse with the Peg during the Wanderer campaign. The part about them "winning and argument with the Archons". I think the one before you recruit the Quarg to help contain the Mereti.

**Comment by Bladewood** [comment]
An argument is not the same as a physical fight...
```
You explain to the Pug that you are here on behalf of the Wanderers: that the region of space the Eye led them to is infested by dangerous robotic automata. They respond, "It is also infested with the pitiable Archons and their pet Quarg. With whom we are not exactly on speaking terms."
	Another of the Pug adds, "To be slightly more precise, we are in fact on blowing-each-other-up terms with them."
	"Why do the Archons hate you?"
	"We had a disagreement recently," says one of the Pug. "We were right. The Archons were wrong. They are still sulking."
```
Can you guys please take this conversation somewhere else?

**Comment by KyroFirehammer** [comment]
Honestly it's fine I'm interested to hear more personally shouldn't the pug be on hodtle terms with the quarg and acherons on code ATM they are only hostile to pirates after FW missions (and drak need to be found! I want to see a t4 ship)

**Comment by warp-core** [comment]
There's no need for the attitudes of the governments to be updated in game as of yet since they never encounter each other so far. Also, being able to take Quarg ships to Pug Iyik through the Eye and have them fight the Pug and repair them (or the Pug ships) to raise reputation isn't something that's really necessary at this point.
As for tier 4 ships, there are already a number of Archons in game that you can encounter if you go exploring although the specifics of these ships and their outfits may or may not be placeholders for future development.
Also, I think what @Bladewood meant was that the topic of this issue has been resolved and that this conversation would be better held elsewhere, perhaps on the Steam forums.

**Comment by KyroFirehammer** [comment]
Actually they can there is a mission in the wanderer storyline where you escort quarg and they can follow you anywhere( even coalition space if you want a punisher)

**Comment by MessyMix** [comment]
Lmao, @KyroFirehammer I think everyone here already knows what you're talking about. This isn't news, mate. That's why @warp-core already addressed the issue, because he said "being able to take Quarg ships to Pug Iyik through the Eye and have them fight the Pug and repair them (or the Pug ships) to raise reputation isn't something that's really necessary at this point."

Read, my dude. He already addressed what you said.

**Comment by KyroFirehammer** [comment]
RIP didn't see it :P also by more drak ships i meant them being more like quarg and having actual ships that arn't living creatures

Also it would be interesting if their was a wormhole in one of the supernovas that was invisible unless you had a specific outfit installed on your ship (similar to the quantum keystones) that the quarg would eventually sell to you (for like 10m a peice)

**Comment by gunqqer** [comment]
Unless we see the ancient Drak that created the Archons (or was it the other way around) we probably won't be seeing any non-living "ships"

**Comment by Disiuze** [comment]
Please discuss this elsewhere, the original discussion is already over and the current one has nothing to do with it.
